lib/modal/ModalCtrl.js

```diff
@@ -0,0 +1,20 @@
+import angular from 'angular';
+import safeApply from '../utils/safe-apply';
+
+var ModalCtrlMod = angular.module('ModalCtrlMod', []);
+
+var ModalCtrl = ModalCtrlMod.controller('ModalCtrl', [
+  '$scope',
+  '$attrs',
+  function($scope, attrs){
+
+    window.addEventListener('keydown', function(e){
+      if ($scope.isOpen && e.keyCode === 27) {
```

Comment:
Maybe bind the keycode to a variable called `ESC` (or I guess `const` might be the best)